src/actions/render.js

```diff
@@ -3,74 +3,85 @@ const Mustache = require("mustache");
 const Result = require("../utils/result");
 const { normalizeYaml, PLURAL, SINGULAR } = require("../actions/normalize");
 const { loadFile } = require("../utils/fileUtils");
-const { groupKeywords, platformKeywords } = require("../model/keywords");
+const { accessiblityKeywords, groupKeywords, platformKeywords } = require("../model/keywords");
 const { groupByKey } = require("../utils/arrayUtils");
 const { flatten } = require("../utils/arrayUtils");
 
 const render = (data, outputPath, platforms, languages) => {
   const translations = normalizeYaml(data, platforms, languages);
-  return languages
-    .reduce((acc, language) => {
-      const translationsForLanguage = translations.filter(
-        t => t.language === language
-      );
+  const translationsPerLanguage = groupByKey(translations, t => t.language);
+
+  const localizationRenders = Object.keys(translationsPerLanguage).reduce((acc, language) => {
+    const translationsForLanguage = translationsPerLanguage[language];
+    const renderResults = platforms.map(platform => {
+      const translations = translationsForLanguage.filter(t => includeInPlatform(t, platform));
+      const view = createLocalizationView(translations, platform);
+      return renderLocalization(view, platform, language, outputPath);
```

Comment:
Consider `renderLocalizationView` to be consistent with `renderCodeGenView` below?